lib/spack/spack/test/versions.py

```diff
@@ -98,160 +106,160 @@ def check_union(expected, a, b):
 
 
 def test_string_prefix():
-    assert_ver_eq("xsdk-0.2.0", "xsdk-0.2.0")
-    assert_ver_lt("xsdk-0.2.0", "xsdk-0.3")
-    assert_ver_gt("xsdk-0.3", "xsdk-0.2.0")
+    assert_ver_eq("=xsdk-0.2.0", "=xsdk-0.2.0")
+    assert_ver_lt("=xsdk-0.2.0", "=xsdk-0.3")
+    assert_ver_gt("=xsdk-0.3", "=xsdk-0.2.0")
```

Comment:
shouldn't we also test that the ranges compare equal?

Reply:
it's tested in `test_version_ranges`